Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM centos:7
+FROM centos:stream8
 
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
-    yum -y install epel-release git && \
-    yum -y install python2-pip && \
+    yum -y install git glibc-langpack-en epel-release && \
```

Comment:
we shouldn't need `epel-release` anymore (pip is in centos these days)

Reply:
Truth. Koji is in epel though.

Reply:
Huh, yeah.